website/docs/r/is_flow_log.html.markdown

```diff
@@ -15,47 +15,47 @@ Create, update, delete and suspend the flow log resource. For more information,
 
 ```terraform
 
-resource "ibm_is_instance" "testacc_instance" {
-  name    = "testinstance"
-  image   = "7eb4e35b-4257-56f8-d7da-326d85452591"
+resource "ibm_is_instance" "example" {
+  name    = "example-instance"
+  image   = ibm_is_image.example.id
   profile = "b-2x8"
 
   primary_network_interface {
     port_speed = "1000"
-    subnet     = "70be8eae-134c-436e-a86e-04849f84cb34"
+    subnet     = ibm_is_subnet.example.id
   }
 
-  vpc  = "01eda778-b822-43a2-816d-d30713df5e13"
+  vpc  = ibm_is_vpc.example.id
   zone = "us-south-1"
-  keys = ["eac87f33-0c00-4da7-aa66-dc2d972148bd"]
+  keys = [ibm_is_ssh_key.example.id]
 }
 
 
-data "ibm_resource_group" "instance_group" {
-  name = var.resource_group
+data "ibm_resource_group" "example" {
+  name = "example-instance-group"
```

Comment:
Suggestion: Can we have pattern of name similar as you have used in others like, `name = "example-resource_group"`